sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java

```diff
@@ -80,6 +80,7 @@ public DefaultAndroidEventProcessor(
     this.context = Objects.requireNonNull(context, "The application context is required.");
     this.options = Objects.requireNonNull(options, "The SentryOptions is required.");
 
+    // TODO: Executor could be ISentryExecutorService
```

Comment:
if we want to reuse ISentryExecutorService, getter SentryOptions.getExecutorService should be public

Reply:
I'd rather we don't make any of this public

Reply:
I'll remove the comment and close this then, was just trying to reuse the executor